CMakeLists.txt

```diff
@@ -100,43 +113,44 @@ add_subdirectory(ext/pybind11)
 # =============================================================================
 # Include path from external libraries
 # =============================================================================
-include_directories(${PROJECT_SOURCE_DIR}/ext/cli11/include)
+include_directories(${NMODL_PROJECT_SOURCE_DIR}/ext/cli11/include)
 
 # =============================================================================
 # Project version from git and project directories
 # =============================================================================
 set(PROJECT_VERSION ${PROJECT_VERSION_MAJOR}.${PROJECT_VERSION_MINOR})
 
 # generate file with version number from git and nrnunits.lib file path
-configure_file(${CMAKE_CURRENT_SOURCE_DIR}/src/config/config.cpp.in
-               ${CMAKE_CURRENT_BINARY_DIR}/config.cpp @ONLY)
+configure_file(${CMAKE_CURRENT_SOURCE_DIR}/src/config/config.cpp.in ${CMAKE_BINARY_DIR}/config.cpp
```

Comment:
This would place the config.cpp in a new place - top level build dir, is that change intended?

Reply:
moved to build/config now